xija/component/heat.py

```diff
@@ -821,30 +821,32 @@ def __str__(self):
     @property
     def par_idxs(self):
         if not hasattr(self, '_par_idxs'):
-            # Make a regex corresponding to the last bit of each power
-            # parameter name.  E.g. "pow_1xxx" => "1...".
-            power_par_res = [par.name[4:].replace('x', '.')
-                             for par in self.power_pars]
-
-            par_idxs = np.zeros(6612, dtype=np.int) - 1
-            for fep_count in range(0, 7):
-                for ccd_count in range(0, 7):
-                    for vid_board in range(0, 2):
-                        for clocking in range(0, 2):
-                            state = "{}{}{}{}".format(fep_count, ccd_count,
-                                                      vid_board, clocking)
-                            idx = int(state)
-                            for i, power_par_re in enumerate(power_par_res):
-                                if re.match(power_par_re, state):
-                                    par_idxs[idx] = i
-                                    break
-                            else:
-                                raise ValueError('No match for power state {}'
-                                                 .format(state))
+            if not hasattr(self, '_par_idxs_static'):
+                print('computing new _par_idxs')
```

Comment:
Unless you think we should leave it in, I'd prefer to get rid of the print statement before merging.

Reply:
No, I didn't intend to leave that.  In fact this whole PR was a bit of just toying around for a use case that may or may not happen.  But I don't see any harm in this as long as someone else has looked at the code.